src/ontology/foodon-edit.ofn

```diff
@@ -19995,6 +19998,33 @@ AnnotationAssertion(dc:date obo:FOODON_00004316 "2022-10-22T23:19:58Z"^^xsd:date
 AnnotationAssertion(rdfs:label obo:FOODON_00004316 "heating water to boiling point"@en)
 SubClassOf(obo:FOODON_00004316 obo:FOODON_00004315)
 
+# Class: obo:FOODON_00004317 (juvenile fish)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004317 "A fish that has not reached to sexual maturity."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004317 "https://en.wikipedia.org/wiki/Juvenile_fish"@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004317 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dc:date obo:FOODON_00004317 "2022-10-28T11:21:32Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004317 "juvenile fish")
+SubClassOf(obo:FOODON_00004317 obo:FOODON_03411222)
+
+# Class: obo:FOODON_00004318 (fry (fish))
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004318 "A fish that is recently hatched and has reached the stage where its yolk-sac has almost disappeared and its swim bladder is operational to the point where the fish can actively feed for itself."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004318 "https://en.wikipedia.org/wiki/Juvenile_fish"@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004318 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dc:date obo:FOODON_00004318 "2022-10-28T11:24:42Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004318 "fry (fish)")
+SubClassOf(obo:FOODON_00004318 obo:FOODON_03411222)
+
+# Class: obo:FOODON_00004319 (fish egg)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004319 "An egg or spawn of a fish that is contained within or separated from the ovary."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004319 "https://en.wikipedia.org/wiki/Juvenile_fish"@en)
+AnnotationAssertion(dc:contributor obo:FOODON_00004319 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dc:date obo:FOODON_00004319 "2022-10-28T11:36:35Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004319 "fish egg")
+SubClassOf(obo:FOODON_00004319 obo:FOODON_03411222)
+
 # Class: obo:FOODON_0002899 (clown featherback)
```

Comment:
BUT if we made it 8 digits it would collide with another existing entry, goby 00002899 !!! 
Actually can we deprecate clown featherback and replace with new NCBITaxon Chitala ornata
http://purl.obolibrary.org/obo/NCBITaxon_112145 ?

Reply:
Posting today's discussion points as a reminder for the future:
This entry is not part of this pull request. But it's good that the taxon is already in FoodOn through the FDA seafood list. 
[https://www.ebi.ac.uk/ols/ontologies/foodon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCBITaxon_112163
](https://www.ebi.ac.uk/ols/ontologies/foodon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCBITaxon_112163
)
- We need to deprecate/fix these old seafood entries.